x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -25,6 +25,7 @@ export interface CreateCaseFlyoutProps {
   onClose?: () => void;
   onSuccess?: (theCase: Case) => Promise<void>;
   attachments?: CaseAttachmentsWithoutOwner;
+  optionalContent?: React.ReactNode;
```

Comment:
Could we rename this prop to `headerContent`? `optionalContent` seems redundant since the `?` indicates that it is optional.